src/main/java/blackjack/BlackjackApplication.java

```diff
@@ -0,0 +1,100 @@
+package blackjack;
+
+import static blackjack.view.OutputView.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import blackjack.domain.player.Dealer;
+import blackjack.domain.card.Deck;
+import blackjack.domain.player.Name;
+import blackjack.domain.player.Player;
+import blackjack.domain.player.Players;
+import blackjack.domain.Selection;
+import blackjack.dto.PlayerDto;
+import blackjack.view.InputView;
+
+public class BlackjackApplication {
```

Comment:
메인 클래스가 너무 복잡하네요.
아래와 같이 수정해보시기 바랍니다.

```suggestion
public class BlackjackApplication {
    public static void main(String[] args) {
        var blackjackGame = new BlackjackGame();
        blackjackGame.run();
    }
}
```

Reply:
Game으로 분리하는 방법도 좋다고 생각합니다. 그런데 Game의 역할이 결국 블랙잭 요구사항에 맞게 도메인과 뷰의 프로세스(순서)를 담당하다고 생각했어요. 이 역할을 볼 땐 BlackjackApplicaiton이라고 명시하고 이 곳 메인에 구현하는 것도 괜찮다고 생각했는데 메인에 직접 구현하는 방법이 안좋은 이유가 있나요?

Reply:
컨트롤러가 영원히 하나인 경우를 가정한다면 메인 클래스에 구현해도 괜찮을 것 같아요.
근데 영원히 하나라는 것을 확신할 수는 없을 것 같아요.
현업에서 요구사항의 변경 속도는 굉장히 빠르고, 예측이 불가능하거든요.
그렇기 때문에 기존 코드를 변경하지 않고 기능을 확장할 수 있는 방법을 생각해보시면 좋을 것 같아요. (이것을 OCP 라고 합니다.)